js/components/base-components/listWithChoice.js

```diff
@@ -0,0 +1,43 @@
+var React = require('react');
+var BaseElement = require('./listItem');
```

Comment:
Can we call this `ListItem`? This name might give the impression that we have a `BaseElement` component.